portfolio/src/main/webapp/script.js

```diff
@@ -53,7 +53,9 @@ function createComment(titleText, bodyText){
 function deleteComments(){
     var init = {method: 'POST'}
     var request = new Request('/delete-data',init);
-    fetch(request).then(displayComment(0));
+    fetch(request).then(() => {
+        displayComment(10);
```

Comment:
why display 10? if we delete all comments, there should be no comments to display so 0 seems to make sense.